lib/amazonka/src/Amazonka/Env.hs

```diff
@@ -84,6 +87,10 @@ env_region f e@Env {region} = f region <&> \region' -> e {region = region'}
 env_logger :: Lens' (Env' withAuth) Logger
```

Comment:
The `env_` naming convention seems a little personal / out of place?

Reply:
This is for consistency with the lenses generated for service bindings cf. https://github.com/brendanhay/amazonka/blob/main/lib/services/amazonka-s3/gen/Amazonka/S3/Lens.hs

If you think we should use a different convention, I'm happy to do that treewide in another PR?

Reply:
The naming for the generated lenses exists because there aren't many obvious conventions that are guaranteed to be unambiguous, not because they're particuarly on brand with Haskell naming conventions.

Consider naming lenses as `camelCase {type-name} <>  camelCase {field-name}`:

```
MyDataType.foo -> myDataTypeFoo
MyData.typeFoo -> myDataTypeFoo
```

Versus `camelCase {type-name} <> "_" <> {verbatim-field-name}`:

```
MyDataType.foo -> myDataType_foo
MyData.typeFoo -> myData_typeFoo
```

I'm fine either way, doesn't need to be changed.

Reply:
I thought this over last weekend, and I think it's better for amazonka to be consistent within itself, even if that means the hand-written stuff is a bit different to the "standard". As with the service bindings, these lenses are for people who don't use `generic-lens` or `generic-optics`.